jekyll/_cci2/local-cli.md

```diff
@@ -7,170 +7,311 @@ categories: [troubleshooting]
 
-### [Alternative] Install on macOS with Homebrew
+#### Install With Homebrew
 {:.no_toc}
 
-If you're using [Homebrew](https://brew.sh/) with macOS, you can install the CLI with the following command:
+If you’re using [Homebrew](https://brew.sh/) with macOS, you can install the CLI with the following command:
 
-```
+```sh
 brew install circleci
 ```
 
-**Note:** If you already have Docker for Mac installed, use `brew install --ignore-dependencies circleci`.
+**Note**: If you already have Docker for Mac installed, use `brew install --ignore-dependencies circleci`.
 
-### [Alternative] Manual download
-{:.no_toc}
+### Manual Download
 
 You can visit the [Github releases](https://github.com/CircleCI-Public/circleci-cli/releases) page for the CLI to manually download and install. This approach is best if you would like the installed CLI to be in a specific path on your system.
 
+## Updating The CLI
 
-## Configuring the CLI
+You can update to the newest version of the CLI using the following command: `circleci update`. If you would just like to check for updates manually (and not install them) use the command: `circleci update check`.
+
+### Updating the Legacy CLI
+{:.no_toc}
 
-Before using the CLI you need to generate a CircleCI API Token from the [Personal API Token tab](https://circleci.com/account/api). Once you have a token configure the CLI by running:
+The newest version of the CLI is a [CircleCI-Public open source project](https://github.com/CircleCI-Public/circleci-cli). If you have the [old CLI installed](https://github.com/circleci/local-cli), run the following commands to update and switch to the new CLI:
 
+```sh
+circleci update 
+circleci switch
 ```
-$ circleci setup
+
+This command may prompt you for `sudo` if your user doesn't have write permissions to the install directory, `/usr/local/bin`.
+
+## Configuring The CLI
+
+Before using the CLI you need to generate a CircleCI API Token from the [Personal API Token tab](https://circleci.com/account/api). Once you have a token, configure the CLI by running:
+
+```sh
+circleci setup
 ```
 
-Setup will prompt you for configuration settings. If you are using the CLI with `circleci.com`, use the default `CircleCI Host`. If you are using CircleCI Enterprise change the value to your installation address (for example, `circleci.my-org.com`).
+Setup will prompt you for configuration settings. If you are using the CLI with circleci.com, use the default CircleCI Host. If you are using CircleCI Enterprise change the value to your installation address (for example, circleci.my-org.com).
 
+## Validate A CircleCI Config
 
-## Validate A CircleCI config
+The CLI can be used to validate your `.circleci/config.yml` file. This can save
+you from pushing commits just for the sake of testing your configuration. 
```

Comment:
for localization, make this a bit more formal language: You can avoid pushing additional commits to test your config.yml by using the CLI to validate your config locally. (save and for the sake of can be confusing)